packages/theme/components/BottomNavigation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+<!-- TODO: create logic with isActive prop for BottomNavigationItems -->
+  <SfBottomNavigation class="navigation-bottom smartphone-only">
+      <SfBottomNavigationItem
+        :class="$route.path == '/' ? 'sf-bottom-navigation__item--active' : ''"
+        icon="home" size="20px" label="Home"
```

Comment:
I think here something is broken with the linter as some attributes are in the same line while other are moved to the next on

Reply:
Ok done.